   This paches moves isolated configrations that should be place in
   EMBEDDED menu.

---
 init/Kconfig |   52 	26 +	26 -	0 !
 1 file changed, 26 insertions(+), 26 deletions(-)

Index: linux-2.6.16-ALP-BRANCH_SS-mips.git/init/Kconfig
===================================================================
--- linux-2.6.16-ALP-BRANCH_SS-mips.git.orig/init/Kconfig
+++ linux-2.6.16-ALP-BRANCH_SS-mips.git/init/Kconfig
@@ -216,32 +216,6 @@ config CPUSETS
 
 source "usr/Kconfig"
 
-config UID16
-	bool "Enable 16-bit UID system calls" if EMBEDDED
-	depends on ARM || CRIS || FRV || H8300 || X86_32 || M68K || (S390 && !64BIT) || SUPERH || SPARC32 || (SPARC64 && SPARC32_COMPAT) || UML || (X86_64 && IA32_EMULATION)
-	default y
-	help
-	  This enables the legacy 16-bit UID syscall wrappers.
-
-config VM86
-	depends X86
-	default y
-	bool "Enable VM86 support" if EMBEDDED
-	help
-          This option is required by programs like DOSEMU to run 16-bit legacy
-	  code on X86 processors. It also may be needed by software like
-          XFree86 to initialize some video cards via BIOS. Disabling this
-          option saves about 6k.
-
-config MEASURE_INLINES
-	default n
-	bool "Enable inline measurement" if EMBEDDED
-	help
-          This marks inline functions deprecated, causing a kernel compile
-          to spew information about the location of each compilation.
-          With some post-processing, this can be used to generate counts
-          and footprints. Say N unless you want a very noisy build.
-
 config CC_OPTIMIZE_FOR_SIZE
 	bool "Optimize for size (Look out for broken compilers!)"
 	default y
@@ -291,6 +265,32 @@ menuconfig EMBEDDED
           environments which can tolerate a "non-standard" kernel.
           Only use this if you really know what you are doing.
 
+config UID16
+	bool "Enable 16-bit UID system calls" if EMBEDDED
+	depends on ARM || CRIS || FRV || H8300 || X86_32 || M68K || (S390 && !64BIT) || SUPERH || SPARC32 || (SPARC64 && SPARC32_COMPAT) || UML || (X86_64 && IA32_EMULATION)
+	default y
+	help
+	  This enables the legacy 16-bit UID syscall wrappers.
+
+config VM86
+	depends X86
+	default y
+	bool "Enable VM86 support" if EMBEDDED
+	help
+          This option is required by programs like DOSEMU to run 16-bit legacy
+	  code on X86 processors. It also may be needed by software like
+          XFree86 to initialize some video cards via BIOS. Disabling this
+          option saves about 6k.
+
+config MEASURE_INLINES
+	default n
+	bool "Enable inline measurement" if EMBEDDED
+	help
+          This marks inline functions deprecated, causing a kernel compile
+          to spew information about the location of each compilation.
+          With some post-processing, this can be used to generate counts
+          and footprints. Say N unless you want a very noisy build.
+
 config KALLSYMS
 	 bool "Load all symbols for debugging/kksymoops" if EMBEDDED
 	 default y
